kitsune/community/management/commands/send_welcome_emails.py

```diff
@@ -62,6 +70,10 @@ def _make_answer_email(locale, to):
 
         # This doesn't need localized, and so don't need the `safe_translation` helper.
         for user in User.objects.filter(id__in=l10n_recipient_ids):
+            # Get team member info for first_l10n emails
+            team_info = get_community_team_member_info('first_l10n')
```

Comment:
You are calling this twice, here and in the `_make_answer_email`. Do we need this in both places?